.devcontainer/Dockerfile

```diff
@@ -1,22 +1,16 @@
-# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.191.0/containers/dotnetcore/.devcontainer/base.Dockerfile
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/dotnet/.devcontainer/base.Dockerfile
```

Comment:
We should no longer point users to microsoft/vscode-dev-containers. The newest repos are the in the `devcontainers` org, with templates in https://github.com/devcontainers/templates.

Reply:
The new images repo is also in the `devcontainers` org, and the images include Dockerfiles: https://github.com/devcontainers/images/tree/main/src/dotnet/.devcontainer.